config/default/kustomization.yaml

```diff
@@ -79,9 +79,4 @@ components:
 #  objref:
 #    kind: Service
 #    version: v1
-#    name: webhook-service
-
-images:
```

Comment:
This images entry was edit by kyma cli, consider as bug, the followup issue created here: https://github.com/kyma-project/cli/issues/1556